app/components/work_packages/date_picker/date_form_component.html.erb

```diff
@@ -18,7 +18,7 @@
         end
 
         start_date.with_row(classes: "wp-datepicker-dialog-date-form--text-field-container") do
-          render(Primer::Alpha::TextField.new(**text_field_options(name: :start_date, label: start_date_label)))
+          render(Primer::Alpha::TextField.new(**text_field_options(name: :start_date, label: start_date_label), "aria-live": "polite"))
```

Comment:
@bsatarnejad FYI, you can also write this as `aria: { live: :polite }`, which might be helpful if you end up adding multiple `aria-` attributes. Rails form/tag helpers would also support writing this as `aria_live`, but VC doesn't appear to transform snake_case to kebab-case.

(No need to change anything for now)